public/css/main.css

```diff
@@ -585,18 +586,66 @@ nav .nav-content {
   align-items: center;
   justify-content: center;
 }
-
-.image-banner-1 {
-  background-image: url("http://placeimg.com/940/480/nature");
-}
-
 .banner-text {
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
   font-size: 40px;
   font-weight: bolder;
-  color: #26a9c3;
-  text-transform: uppercase;
+  color: white;
+  text-align: center;
+  font-family: 'Lexend Deca', sans-serif;
+}
+/* Volunteer Application Page Styles*/
```

Comment:
Is this code from the components or code specific to the application page?

CSS code for the components will be put into this file and kept up-to date but some of this looks like it's specific to the volunteer-application.html page... CSS specific to that page should go in `/public/css/volunteer-application.css